cg/meta/upload/nipt/nipt.py

```diff
@@ -0,0 +1,105 @@
+"""NIPT ftp upload API"""
+import datetime as dt
+import logging
+from pathlib import Path
+from typing import Optional
+
+from alchy import QueryModel
+
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.constants import Pipeline
+from cg.exc import HousekeeperFileMissingError
+from cg.models.cg_config import CGConfig
+from cg.store import Store, models
+from cg.utils import Process
+from housekeeper.store import Store
+
+LOG = logging.getLogger(__name__)
+
+
+class NiptUploadAPI:
+    """API for NIPT upload via ftp"""
```

Comment:
I think it should be more clear to what we are uploading

Reply:
fixed in d1e79180d6dac13d8d76a5a28085b6907a7ca4dc

Reply:
I think this is better. Then we can elaborate in next PR when we add upload-to-NIPTool-functionality